### DISCUSSION TITLE
Nixie or EasyTouch

### DISCUSSION THREAD

**Discussion by jamestalmage** [tagyoureit/nodejs-poolController#726] [Q&A] [answered]
Other than cost, is there a significant advantage to going the DIY Nixie route vs EasyTouch?

The Nixie option doesn't scare me (well within my skill wheelhouse), but I am also not keen to make pool automation a part-time gig. 

I'm assuming there are automations that an EasyTouch just can't do compared to DIY. Any quick insights into what those are?

**Comment by rstrouse** [comment, marked as answer]
Nixie is a much more capable controller than the EasyTouch.  First, there is no way I would buy an EasyTouch controller.  That was first released around 2005 and really hasn't changed much.  Why they are still selling it is a mystery.  IntelliCenter is a much better choice if you can get your hands on one but even with that Nixie has more capabilities.  When you break down the costs for connectivity and add ons the price differential is not that great between the two.

Primarily, there really are no expansion limits on Nixie.  If you need 64 circuits and 20 pumps from various manufacturers then so be it.  It is also capable of an unlimited number of schedules, valves, and heaters (actually there is a limit but it is the size of a 32 bit integer).  At some point it becomes overkill but many run into the limitations of EasyTouch.

For the most part, you will find one of the biggest drawbacks of EasyTouch is that is is constrained to the current equipment sets.  When new equipment comes out, the support is stuck at whatever the firmware was when you purchased.  The perfect example of this includes the new heater functions from Pentair.  If you have an EasyTouch the support for is incomplete.  Even if you buy an new motherboard, the firmware hasn't been updated in a while.

IntelliCenter does solve some of that but the latest firmware was a disaster.  I don't believe it to be a limitation of the hardware, because they have had struggles with prior versions as well.  At one point njsPC had code in it to fix a schedule reset problem where when a schedule kicked on and it would uninstall the heater mode from the body.  You would think that your solar heater was heating your pool only to find out it had be shut down some time during the week.  It went on like that for half a year.

njsPC does add quite a bit of functionality to any Pentair OCP though.  For the most part they are a closed system but njsPC allows for the functionality to be added to home automation platforms and gives you the ability to automat through a whole host of external services.

**Comment by jamestalmage** [comment]
Perfect answer. Nixie it is!
